OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -264,7 +264,7 @@ public Barrel CheckFire(Actor self, IFacing facing, Target target)
 		public bool ShouldExplode(Actor self) { return !IsReloading; }
```

Comment:
Could this be farmed off to a public virtual method that the (explicit) interface impl wraps?